### ISSUE TITLE
Suggestions on what to improve

### ISSUE THREAD

**Issue by pintariching** [pintariching/rustle#62]
The latest library release adds support for nested component and passing properties to them. A lot of functionality still has to be tested by expanding the tests. 

Feel free to try out the compiler with everything you can throw at it so we can see what still has to be fixed!

**Comment by qwertydelle** [comment]
Does css work now?

**Comment by pintariching** [comment]
Yes, I think it works. It generates an `app.css` file that you link in your `index.html` and it dumps all the CSS between the `style` tags appended with a random string into it.

**Comment by qwertydelle** [comment]
Gonna try and find what I can add or improve this weekend.
